programs/marinade-finance/src/lib.rs

```diff
@@ -719,10 +713,11 @@ impl<'info> DerefMut for UpdateActive<'info> {
 pub struct UpdateDeactivated<'info> {
     pub common: UpdateCommon<'info>,
 
+    /// CHECK: not important
```

Comment:
TODO: check how this is used. Is it VaLV7StdK6A3KfqyePbHyaBefjeibq9gakECFtXNM4m? the bot account